src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -206,6 +206,154 @@ def test_WINNF_FT_S_SIQ_8(self):
     self.assertEqual(response['response']['responseCode'], 103)
 
   @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_9(self):
+    """Multiple SIQ requests as claimed PPAs or as GAAs.
+
+    Response Code must be 0 for all CBSDs.
+    """
+
+    # Load 4 devices
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    device_f = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_f.json')))
+
+    # First PPA with device_a and FR1 = 3550 - 3560
+    pal_low_frequency1 = 3550000000
+    pal_high_frequency1 = 3560000000
+    pal_record1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+    ppa_record1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+    ppa_record1, pal_record1 = makePpaAndPalRecordsConsistent(
+        ppa_record1, [pal_record1], pal_low_frequency1, pal_high_frequency1,
+        device_a['userId'])
+
+    # Move device_a into the first PPA zone
+    device_a['installationParam']['latitude'], device_a['installationParam'][
+        'longitude'] = getRandomLatLongInPolygon(ppa_record1)
+
+    # Second PPA with device_c and FR2 = 3600 - 3610
+    pal_low_frequency2 = 3600000000
+    pal_high_frequency2 = 3610000000
+    pal_record2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_1.json')))
+    ppa_record2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_1.json')))
+    ppa_record2, pal_record2 = makePpaAndPalRecordsConsistent(
+        ppa_record2, [pal_record2], pal_low_frequency2, pal_high_frequency2,
+        device_c['userId'])
+
+    # Move device_c into the second PPA zone
+    device_c['installationParam']['latitude'], device_c['installationParam'][
+        'longitude'] = getRandomLatLongInPolygon(ppa_record2)
+
+    # Register 4 devices.
+    cbsd_ids = self.assertRegistered([device_a, device_c, device_e, device_f])
+
+    # Update PPA record with device_a's CBSD ID and Inject data
+    ppa_record1['ppaInfo']['cbsdReferenceId'] = [cbsd_ids[0]]
+    self._sas_admin.InjectPalDatabaseRecord(pal_record1[0])
+    zone_id = self._sas_admin.InjectZoneData({'record': ppa_record1})
+    self.assertTrue(zone_id)
+
+    # Update PPA record with device_c's CBSD ID and Inject data
+    ppa_record2['ppaInfo']['cbsdReferenceId'] = [cbsd_ids[1]]
+    self._sas_admin.InjectPalDatabaseRecord(pal_record2[0])
+    zone_id = self._sas_admin.InjectZoneData({'record': ppa_record2})
+    self.assertTrue(zone_id)
+
```

Comment:
I think we need to trigger the daily activity to activate the PPA, right?

Reply:
Reda, 
In my understanding we trigger daily activities only if the PPA is being protected. Here, we're just looking to mark the channel as "PAL", so no need to trigger.

Reply:
thank you Harini for your response, 
In my understanding the SAS may not consider the impact of the PAL on the spectrum channels unless it runs the IAP.
and also in the SIQ.5 we can see that it triggers the daily activities, right?

Reply:
Reda, it was the decision of WinnForum that you need IAP ("daily activities"/"CPAS") to trigger *protection* of a PPA but that you can get a PAL channel earlier than that.

Reply:
thank you Harini , you are right so we need to update SIQ.5.